transactionparser/src/channeldetails.js

```diff
@@ -0,0 +1,38 @@
+const normalizeNodes = require('./utils').normalizeNodes;
+const dropsToXRP = require('./utils').dropsToXRP;
+const BigNumber = require('bignumber.js');
+
+function summarizePaymentChannel(node) {
+
+      const final = (node.diffType === 'CreatedNode') ? node.newFields : node.finalFields;
+      const prev = node.previousFields || {};
+
+      return {
+        channelId: node.ledgerIndex,
+        source: final.Account,
+        destination: final.Destination,
+        channel_amount_change: prev.Amount ? 
```

Comment:
What do you think about including the diffs of all changed fields (similar to [parseBalanceChanges](https://github.com/ripple/ripple-lib-extensions/tree/master/transactionparser#parsebalancechangesmetadata)) in a `changes` field?

Reply:
What other changed fields are there?